proxyd/config.go

```diff
@@ -36,11 +36,24 @@ type CacheConfig struct {
 	TTL     TOMLDuration `toml:"ttl"`
 }
 
+type RedisClientChoice string
+
+const (
+	DefaultChoice RedisClientChoice = "default" // single node
+	ClusterChoice RedisClientChoice = "cluster"
+)
+
 type RedisConfig struct {
-	URL              string `toml:"url"`
-	Namespace        string `toml:"namespace"`
-	ReadURL          string `toml:"read_url"`
-	FallbackToMemory bool   `toml:"fallback_to_memory"`
+	// If choice = "cluster" is specified (toml), you can specify url string for multi-node cluster:
+	//    "redis://<user>:<password>@<host>:<port>?addr=<host2>:<port2>&addr=<host3>:<port3>"
+	// OR "rediss://<user>:<password>@<host>:<port>?addr=<host2>:<port2>&addr=<host3>:<port3>"
+	//
+	// Otherwise, it is also possible to specify single url for Redis cluster with proxy support.
+	URL              string            `toml:"url"`
+	Namespace        string            `toml:"namespace"`
+	ReadURL          string            `toml:"read_url"`
+	FallbackToMemory bool              `toml:"fallback_to_memory"`
+	Choice           RedisClientChoice `toml:"choice"`
```

Comment:
Could choice be updated to be a boolean since there are only two options, and can it be renamed to RedisCluster

Reply:
Done 👌!